package.json

```diff
@@ -94,9 +94,9 @@
     "yargs": "^15.4.1"
   },
   "devDependencies": {
-    "@types/jest": "^24.0.15",
+    "@types/jest": "28.1.1",
```

Comment:
bump these deps to align with tag `ts3.9`, but pin version instead of tag, since tag lockfile updates are less reliable